docs/messages_guide/src/asciidoc/_chapters/ddl_msgs.adoc

```diff
@@ -254,6 +254,24 @@ local super ID can perform a replication operation for a catalog and for
 multiple schemas in a single catalog. Additionally, the schema owner can
 perform a replication operation for a single schema.
 
+[[SQL-1018]]
+== SQL 1018
+
+```
+Grant of role or privilege <privilege> from <user-or-role-1> to <user-or-role-2> not found, revoke request ignored.
+```
+
+Where <privilege> is a {project-name} privilege and <user-or-role-1> and <user-or-role-2> are user IDs or roles.
+
```

Comment:
user-or-role-2 can be the special role PUBLIC but user-or_role-1 cannot be PUBLIC
You might want to call the <authorization IDs> like other messages.